src/Uno.UI/UI/Xaml/Input/PointerRoutedEventArgs.wasm.cs

```diff
@@ -101,7 +102,7 @@ private static ulong ToTimeStamp(double timestamp)
 		{
 			if (!_bootTime.HasValue)
 			{
-				_bootTime = (ulong) (double.Parse(WebAssemblyRuntime.InvokeJS("Date.now() - performance.now()")) * TimeSpan.TicksPerMillisecond);
+				_bootTime = (ulong)(double.Parse(WebAssemblyRuntime.InvokeJS("Date.now() - performance.now()"), CultureInfo.InvariantCulture) * TimeSpan.TicksPerMillisecond);
```

Comment:
Is there a way to do interop in binary instead of parsing a string like that?

Reply:
Good point, I'll add this as part of https://github.com/unoplatform/uno/issues/2116